site/_layouts/doc.liquid

```diff
@@ -220,7 +220,8 @@
                       <li {% if page.url == "/docs/v1/tech/oauth/" %}class="active"{% endif %}><a href="/docs/v1/tech/oauth/">Overview</a></li>
                       <li {% if page.url == "/docs/v1/tech/oauth/endpoints.html" %}class="active"{% endif %}><a href="/docs/v1/tech/oauth/endpoints">Endpoints</a></li>
                       <li {% if page.url == "/docs/v1/tech/oauth/tokens.html" %}class="active"{% endif %}><a href="/docs/v1/tech/oauth/tokens">Tokens</a></li>
-                        <li {% if page.url == "/docs/v1/tech/core-concepts/modes.html" %}class="active"{% endif %}><a href="/docs/v1/tech/core-concepts/modes">OAuth Modes</a></li>
+                      <li {% if page.url == "/docs/v1/tech/core-concepts/modes.html" %}class="active"{% endif %}><a href="/docs/v1/tech/core-concepts/modes">OAuth Modes</a></li>
```

Comment:
Looks like this is in `core-concepts` but it shows up in the OAuth menu nav.  This is probably just an old error, want to move it?

Reply:
This is from something I did. I wanted to reconcile the nav and the url structure once.

Reply:
Moved the page. Still need to get redirect working. Posted in Slack.

Reply:
please don't move the page. I want to do a large nav overhaul and want to move it at that time.